package.json

```diff
@@ -7,6 +7,9 @@
   "author": "ECMA TC39",
   "license": "SEE LICENSE IN https://tc39.github.io/ecma262/#sec-copyright-and-software-license",
   "homepage": "https://tc39.github.io/proposal-class-fields/",
+  "scripts": {
+    "build": "mkdir out && ecmarkup spec.html out/index.html"
```

Comment:
```suggestion
    "prebuild": "mkdir out",
    "build": "ecmarkup spec.html out/index.html"
```

Reply:
Oh for crying out loud, it fails when it exists and I'm too stupid to have checked. Thanks @ljharb . I hate to make it two steps, but I can't think of a command that would be both \*nix and Windows friendly to do it as a one-off.

I feel a PR to ecmarkup coming on..., but in the meantime, any ideas other than the two-script solution or is this the best we can do?

`deploy.sh` happily used `mkdir out && ...`, I guess we know that `out` won't exist? (I've never used Travis.) So `deploy.sh` use `npm run prebuild && npm run build`...

Reply:
I find more scripts inherently superior, since it’s more unixlike, so this is the best I’d want :-)

Also no, npm runs pre scripts for you. You only need npm run build

Reply:
no, but you can use the rimraf module and mkdirp module to handle things.

Reply:
Done! But do you have a clever trick to avoid that `&&` in `prebuild`? :-) (I was really, really hoping `preprebuild` would work, but alas...)

Reply:
you'd have to do something excessive like:
```
"clean": "rimraf out",
"postclean": "mkdir out",
"prebuild": "npm run clean",
"build": "ecmarkup whatever",
```

but `&&` works in windows afaik.

Reply:
Yup. The semantics are slightly different (it always runs the second command, even if the first fails) but it works. So I think we're finally there. I **really** appreciate all the feedback and help.